website/docs/r/shield_protection.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "aws"
+page_title: "AWS: aws_shield_protection"
+sidebar_current: "docs-aws-resource-shield-protection"
+description: |-
+  Enables AWS Shield Advanced for a specific AWS resource.
+---
+
+# aws_shield_protection
+
+Enables AWS Shield Advanced for a specific AWS resource.
+The resource can be an Amazon CloudFront distribution, Elastic Load Balancing load balancer, AWS Global Accelerator accelerator, Elastic IP Address, or an Amazon Route 53 hosted zone.
+
+## Example Usage
+
+### Create protection
+
+```hcl
+variable "name" {
+  default = "bar"
+}
+
+data "aws_availability_zones" "available" {}
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
+resource "aws_eip" "foo" {
```

Comment:
For AWS accounts that still support EC2-Classic in a region, this resource configuration needs to include `vpc = true`.